packages/mongodb-chatbot-evaluation/src/generate/generateLlmQuizQuestionAnswer.ts

```diff
@@ -0,0 +1,176 @@
+import { QuizGeneratedData } from "./GeneratedDataStore";
+import { ChatLlm, ObjectId, UserMessage } from "mongodb-chatbot-server";
+import { logger } from "mongodb-rag-core";
+import { GenerateDataFunc } from "./GenerateDataFunc";
+import {
+  QuizQuestionTestCase,
+  QuizQuestionTestCaseData,
+  SomeTestCase,
+  isQuizQuestionTestCase,
+} from "./TestCase";
+import { sleep } from "../utils/sleep";
+import assert from "assert/strict";
+
+export interface MakeGenerateQuizDataParams {
+  /**
+    Few-shot examples of quiz questions
+    @example
+    [
+      {
+        questionText: "What's the best Italian food?",
+        answers: [
+          { label: "A", answer: "Tacos", isCorrect: false },
+          { label: "B", answer: "Pizza", isCorrect: true },
+          { label: "C", answer: "Sushi", isCorrect: false },
+          { label: "D", answer: "Enchiladas", isCorrect: false },
+        ],
+      },
+    ]
+   */
+  quizQuestionExamples?: QuizQuestionTestCaseData[];
+
+  /**
+    Few words of the subject of the quiz questions. Included in the prompt.
+    @example "Food"
+   */
+  subject?: string;
+
+  /**
+    The large language model to use for generating conversation data.
+   */
+  chatLlm: ChatLlm;
+
+  /**
+    The name of the language model to use for generating the quiz question answers.
+   */
+  modelName: string;
+
+  /**
+    Number of milliseconds to sleep between each conversation generation.
+    Helpful for rate limiting.
+   */
+  sleepMs?: number;
+}
+
+/**
+  Generate answer to multiple-choice quiz questions using a large language model (LLM).
+
+  This can be useful for evaluating how an LLM performance on the subject matter of the multiple choice questions.
+
+  The prompt is based on this blog post from Hugging Face: https://huggingface.co/blog/open-llm-leaderboard-mmlu
+  It follows the [HELM prompt format](https://huggingface.co/blog/open-llm-leaderboard-mmlu#mmlu-comes-in-all-shapes-and-sizes-looking-at-the-prompts).
```

Comment:
Would it be worth splitting this out into a reusable `makeHelmPrompt()` function w/ accompanying tests? Not necessary for this PR but might be useful down the line + as documentation of the format.

Reply:
HELM is just for quiz-style questions, the current `makeQuizQuestionPrompt()` does what you're describing. i can refactor comments a bit to make this clearer + also export this function and test it